tasks/main.yml

```diff
@@ -32,46 +32,6 @@
   tags:
   - oc
 
-# Setup policy binding
-- name: query policybinding
-  command: oc get policybinding {{ osbs_namespace }}:default --namespace {{ osbs_namespace }}
-  environment: "{{ osbs_environment }}"
-  register: policybinding_query
-  failed_when: policybinding_query.rc != 0 and ('not found' not in policybinding_query.stderr)
-  changed_when: false
-  when: osbs_is_admin
-  tags:
-  - oc
-
-- name: create a policybinding
```

Comment:
Isn't this needed initially? I've had problems creating roles/rolebindings if this is not called first.

Reply:
It might be ok because we're not binding to project roles, but instead cluster roles.

Reply:
We are binding this to a project namespace, but a new policybinding is not required for that - as a new custom role is not created anymore